docs/developer-docs/latest/setup-deployment-guides/configurations.md

```diff
@@ -1575,3 +1575,108 @@ module.exports = () => ({
   },
 });
 ```
+
+### Role-Based Access Control <BronzeBadge withLinkIcon link="https://strapi.io/pricing" />
+
+Role-Based Access Control (RBAC) is an approach to restricting access to some users. In a Strapi application, users of the admin panel are administrators, and you can easily [configure their roles and permissions](/user-docs/latest/users-roles-permissions/configuring-administrator-roles.md). In addition to the 3 default roles (author, editor, and super admin), you can create custom conditions to match specific needs of roles and permissions management. This requires an EE license with a Bronze plan.
+
+::: warning 🚧 This API is considered unstable for now.
+<br>
+:::
+
+#### Defining a new condition
+
+Conditions are defined as an array of objects. Each condition object can have 4 possible properties:
+
+- `displayName`: the user name displayed in a friendly format, as showed in the admin panel
+- `name`: the condition name, kebab-cased
+- `plugin`: the plugin name, kebab-cased, if your condition is based on a plugin (e.g `content-manager`)
```

Comment:
`if your condition is based on a plugin` or `if your condition is declared into a plugin`?
I think it's 2 completely different meanings.

Reply:
Indeed, and not sure what was the meaning behind this part in the draft. @alexandrebodin might help on this :-)

Reply:
@petersg83 / @Convly Would you mind explaining a bit more what this `plugin` is used for? (and if it's really optional). Thanks 🙏

Reply:
Yep :)
It is the plugin that created the condition.
It should be let undefined if it is not added from a plugin (directly by the user then)

Related code : https://github.com/strapi/strapi/blob/master/packages/strapi-admin/domain/condition.js